### PR TITLE
Adding the customization_script retrieve operation via the REST API

### PR DESCRIPTION
**Pull request by igorrran**
This PR is able to:
* Add a new operation to retrieving a configuration pattern from customization scripts endpoint

### CODE REVIEW

**Comment by abellotti** [comment]
Thanks @igorrran for making this separate PR!!  could you add the collection test in spec/requests/collection_specs.rb and add support for the bulk "query" action (see other collections you'd need a post for that, and a second bulk query test also in spec/requests/collection_specs.rb).

Thanks !!

**Comment by abellotti** [comment]
Ping @jntullo when you get a chance to review 👀   Thanks!!

File: spec/requests/customization_script_spec.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        api_basic_authorize
+        get api_customization_script_url(nil, config_pattern)
+
+        expect(response).to have_http_status(:forbidden)
+        expect(response.parsed_body["error"]).to include("kind" => "forbidden")
+      end
+    end
+
+    context "with valid id" do
+      it "returns both id and href" do
```
Comment (jntullo) [inline spec/requests/customization_script_spec.rb]: just a NIT here, but it should return the whole config pattern resource, not just id and href, so this is a little misleading

Comment (igorrran) [reply spec/requests/customization_script_spec.rb]: Should return all config pattern details(href, id, name, manager_id, created_at, description...)?

Comment (jntullo) [reply spec/requests/customization_script_spec.rb]: @igorrran don't worry about testing all of the details...just update this to describe it as "returns the resource" or something similar. Thanks!!

**Comment by abellotti** [comment]
Thanks @igorrran for making the update and the API Enhancement!! 👍

**Pull request merged**